File: AddressBook.Core/DataAccess/ReportModels.cs
using System.ComponentModel.DataAnnotations;

namespace AddressBook.Core.DataAccess
{
  public class PhoneListReportModel
  {
    public int PersonId { get; set; }

    [Display(ResourceType = typeof (Resources), Name = "PhoneList_PersonName")]
    public string PersonName { get; set; }

    public int PhoneId { get; set; }

    [Display(ResourceType = typeof(Resources), Name = "Phone_PhoneNumber")]
    public string PersonPhone { get; set; }
  }

  public class PhoneCountReportModel
  {
    public int PersonId { get; set; }

    [Display(ResourceType = typeof (Resources), Name = "PhoneList_PersonName")]
    public string PersonName { get; set; }

    [Display(Name = "Телефоны")]
    public int Phones { get; set; }
  }
}

File: AddressBook.Ang/Scripts/spa/Services/ReportModels.ts
module AddressBookApp {
  export interface IPhoneList extends ng.resource.IResource<IPhoneList> {
    personId: number;
    personName: string;
    phoneId: number;
    personPhone: string;
  }
  export interface IPhoneCount extends ng.resource.IResource<IPhoneCount> {
    personId: number;
    personName: string;
    phones: number;
  }

}